pokemon-checker/src/DataTransferObjects/PokemonDTO.ts

```diff
@@ -69,9 +104,28 @@ class PokemonDTO {
     this.baseStats = this.calculateBaseStats();
   }
 
+  /**
+   * Returns Pokemon data in consumable form (strings) for a calling view.
+   */
+  public getDisplayStats(): pokemonDisplayObj {
```

Comment:
`PokemonDisplay.tsx` was relying on the raw object as an `any` type to display, and was not able to use `number` properties when rendering. I've created this method in the DTO so it can provide readable stats to whatever view is calling on it.